maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/RepOrderController.java

```diff
@@ -145,6 +147,30 @@ public ResponseEntity<Object> findByRepIdAndVehicleOwner(@PathVariable int repId
         ));
     }
 
+    @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(description = "Create a rep order record")
+    @ApiResponse(responseCode = "200",
```

Comment:
Many of these duplicated boiler plate annotations can be replaced with 
[StandardApiResponse.java](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/9bade7c6c2b82ec82067c67074c7c890272690e8/maat-court-data-api/src/main/java/gov/uk/courtdata/annotation/StandardApiResponse.java)

See [ConcorContributionsRestController.java](https://github.com/ministryofjustice/laa-maat-court-data-api/blob/0e4ca828653a95453747bc1e6fedb2b2988a7a1b/maat-court-data-api/src/main/java/gov/uk/courtdata/dces/controller/ConcorContributionsRestController.java) for an example

Reply:
I've kept in the 200 response annotations here, as they appear to be unique to each of the methods (with their own operation description). However I've replaced the 400 and 500 ApiResponses in this controller with the StandardApiResponse annotation to reduce some of that boilerplate.